src/mongoose_hooks.erl

```diff
@@ -267,7 +267,7 @@ failed_to_store_message(Acc) ->
     Result :: drop | filter_packet_acc().
 filter_local_packet(FilterAcc = {_From, _To, Acc, _Packet}) ->
     HostType = mongoose_acc:host_type(Acc),
-    run_hook_for_host_type(filter_local_packet, HostType, FilterAcc, []).
+    run_hook_for_host_type(filter_local_packet, HostType, FilterAcc, #{args => []}).
```

Comment:
use ejabberd_hooks interface to add legacy args, we don't want to have a spaghetti code even if we plan to get rid of this code in the nearest future.

Reply:
done